source/_layouts/base.liquid

```diff
@@ -33,50 +32,70 @@
         rel="stylesheet" />
     </noscript>
 
-    <!-- Old version of analytics (https://developers.google.com/analytics/devguides/collection/gajs/). -->
-    <script>
-      var _gaq = _gaq || [];
-      _gaq.push(['_setAccount', 'UA-535380-8']);
-      _gaq.push(['_trackPageview']);
-
-      (function() {
-        var ga = document.createElement('script');
-        ga.type = 'text/javascript';
-        ga.async = true;
-        ga.src = (
-          'https:' == document.location.protocol
-            ? 'https://ssl'
-            : 'http://www'
-        ) + '.google-analytics.com/ga.js';
-        var s = document.getElementsByTagName('script')[0];
-        s.parentNode.insertBefore(ga, s);
-      })();
-    </script>
-
-    <!-- New version of analytics (https://developers.google.com/analytics/devguides/collection/analyticsjs/). -->
+    <!-- Google Analytics (https://developers.google.com/analytics/devguides/collection/gtagjs). -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-535380-14"></script>
     <script>
-      (function(i, s, o, g, r, a, m) {
-        i['GoogleAnalyticsObject'] = r;
-        i[r] = i[r] || function() {
-          (i[r].q = i[r].q || []).push(arguments)
-        },
-        i[r].l = 1 * new Date();
-        a = s.createElement(o),
-        m = s.getElementsByTagName(o)[0];
-        a.async = 1;
-        a.src = g;
-        m.parentNode.insertBefore(a, m)
-      })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag('js', new Date());
 
-      ga('create', 'UA-535380-14', 'sass-lang.com');
-      ga('send', 'pageview');
+      gtag('config', 'UA-535380-14');
     </script>
   </head>
 
   <body data-layout="body">
-    {{ content }}
+    <!--[if lt IE 9]>
+    <p class="browserupgrade">
+    You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.
+    </p>
+    <![endif]-->
+
+    {% include 'header' %}
```

Comment:
Something to document: is there a pattern for when to use include vs renderFile?

Reply:
Good question. `include` can only include other liquid files, and they are parsed as liquid. `renderFile` can include files of any type, and can parse them as any language. Maybe we should just always use `renderFile`? The only downside is that it requires a full path, instead of always using the `_includes` folder. @mirisuzanne What do you think?

Reply:
Hmm, yeah. Not sure I have a strong sense either. Seems like 'include' is nicer, when it works. So maybe just a matter of documenting when to not use it?

Reply:
@mirisuzanne Oh, the other difference is that `includes` have access to all the data/context of their parent scope. `renderFile` only has access to `eleventy` and `page` objects, but you can pass anything else in.

Except... I can't get the "pass data in" part of it to actually work (for the new `alerts` data). I'll stick with `include` for now in that case, but I'll file an issue with 11ty -- apparently `include` is deprecated anyway, so we should move away from it.

Reply:
(Or maybe I'll switch to the newer liquid `{% render %}` tag, if that works...)

Reply:
I was able to get `renderFile` to work, but I'll stick with the more concise `render` for cases where non-liquid parsing isn't required. I'm not sure where to document this usage at this point?

Reply:
I'm not sure either - but now with 3 options (render, renderFile, include) it seems like we should find somewhere?

Reply:
Added in https://github.com/oddbird/sass-site/commit/ba53435ab3b9800be413c15cabe9f6ed31dde533 -- what do you think?

Reply:
That's great. Might be useful to add 'advice' to that in addition to the details of how it works. But I don't think that needs to hold things up here.

Reply:
Sure -- but I'm not sure I actually have advice on this (yet) 😅